rocketpy/plots/rocket_plots.py

```diff
@@ -104,6 +105,181 @@ def thrust_to_weight(self):
+            [0, -y_last],
+            color=vis_args["body"],
+            linewidth=vis_args["line_width"],
+        )
+
+        # Draw nosecone
+        nosecones = self.rocket.aerodynamic_surfaces.get_tuple_by_type(NoseCone)
+        for nose, position in nosecones:
+            x_nosecone = -csys * nose.shape_vec[0] + position
+            y_nosecone = nose.shape_vec[1]
+
+            plt.plot(
+                x_nosecone,
+                y_nosecone,
+                color=vis_args["nose"],
+                linewidth=vis_args["line_width"] - 0.05,
+            )
+            plt.plot(
+                x_nosecone,
+                -y_nosecone,
+                color=vis_args["nose"],
+                linewidth=vis_args["line_width"] - 0.05,
+            )
+
+        # Draw transitions
+        tails = self.rocket.aerodynamic_surfaces.get_tuple_by_type(Tail)
+        for tail, position in tails:
+            x_tail = -csys * tail.shape_vec[0] + position
+            y_tail = tail.shape_vec[1]
+
+            plt.plot(
+                x_tail,
+                y_tail,
+                color=vis_args["tail"],
+                linewidth=vis_args["line_width"],
+            )
+            plt.plot(
+                x_tail,
+                -y_tail,
+                color=vis_args["tail"],
+                linewidth=vis_args["line_width"],
+            )
+
+        # Draw fins
+        fins = self.rocket.aerodynamic_surfaces.get_tuple_by_type(Fins)
+        for fin, position in fins:
+            x_fin = -csys * fin.shape_vec[0] + position
+            y_fin = fin.shape_vec[1] + self.rocket.radius
+
+            plt.plot(
+                x_fin,
+                y_fin,
+                color=vis_args["fins"],
+                linewidth=vis_args["line_width"],
+            )
+            plt.plot(
+                x_fin,
+                -y_fin,
+                color=vis_args["fins"],
+                linewidth=vis_args["line_width"],
+            )
+
+        # Draw rail buttons
+        buttons, pos = self.rocket.rail_buttons[0]
+        lower = pos
+        upper = pos + buttons.buttons_distance * csys
+        plt.scatter(
+            lower, -self.rocket.radius, marker="s", color=vis_args["buttons"], s=10
+        )
+        plt.scatter(
+            upper, -self.rocket.radius, marker="s", color=vis_args["buttons"], s=10
+        )
+
+        # Draw center of mass and center of pressure
+        cm = self.rocket.center_of_mass(0)
+        plt.scatter(cm, 0, color="black", label="Center of Mass", s=30)
+        plt.scatter(cm, 0, facecolors="none", edgecolors="black", s=100)
+
+        cp = self.rocket.cp_position
+        plt.scatter(cp, 0, label="Center Of Pressure", color="red", s=30, zorder=10)
+        plt.scatter(cp, 0, facecolors="none", edgecolors="red", s=100, zorder=10)
+
+        # Set plot attributes
+        plt.title(f"Rocket Geometry")
+        plt.ylim([-self.rocket.radius * 4, self.rocket.radius * 6])
```

Comment:
This code line may be improved in the future to take the "maximum semi span" of the rocket, usually the sum of the fin span and  the rocket radius. But at least it is working for now.